electronicparsers/openmx/parser.py

```diff
@@ -717,6 +734,27 @@ def parse(self, mainfile: str, archive: EntryArchive, logger):
             name='OpenMX', version=str(mainfile_parser.get('program_version'))
         )
 
+        stress_tensor_enabled = mainfile_parser.get('scf.stress.tensor')
+        if stress_tensor_enabled == "on":
+            mainfile_dir = path.dirname(mainfile)
+            stdout_path = None
```

Comment:
Add a comment here:
_Stress tensor values are not written to outfile but only to stdout. that means we have to check if the stdout is actually present and look for the stress values there._